fwm-docs-qs/azuredeploy.json

```diff
@@ -32,95 +32,115 @@
   "resources": [
     {
       "type": "Microsoft.Network/virtualWans",
-      "apiVersion": "2019-08-01",
+      "apiVersion": "2020-06-01",
       "name": "VWan-01",
       "location": "[parameters('location')]",
       "properties": {
         "disableVpnEncryption": false,
         "allowBranchToBranchTraffic": true,
-        "allowVnetToVnetTraffic": false,
         "office365LocalBreakoutCategory": "None",
         "type": "Standard"
       }
     },
     {
       "type": "Microsoft.Network/virtualHubs",
-      "apiVersion": "2020-04-01",
+      "apiVersion": "2020-06-01",
       "name": "Hub-01",
       "location": "[parameters('location')]",
       "dependsOn": [
         "[resourceId('Microsoft.Network/virtualWans', 'VWan-01')]"
       ],
       "properties": {
-        "virtualNetworkConnections": [
-          {
-            "name": "hub-spoke",
-            "properties": {
-              "remoteVirtualNetwork": {
-                "id": "[resourceId('Microsoft.Network/virtualNetworks', 'Spoke-01')]"
-              },
-              "allowHubToRemoteVnetTransit": true,
-              "allowRemoteVnetToUseHubVnetGateways": false,
-              "enableInternetSecurity": true
-            }
-          }
-        ],
         "addressPrefix": "10.1.0.0/16",
         "virtualWan": {
           "id": "[resourceId('Microsoft.Network/virtualWans', 'VWan-01')]"
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Network/virtualHubs/hubVirtualNetworkConnections",
+      "apiVersion": "2020-06-01",
+      "name": "Hub-01/hub-spoke",
+      "dependsOn": [
+        "[resourceId('Microsoft.Network/azureFirewalls', 'AzfwTest')]",
+        "[resourceId('Microsoft.Network/virtualHubs', 'Hub-01')]",
+        "[resourceId('Microsoft.Network/virtualNetworks', 'Spoke-01')]",
+        "[resourceId('Microsoft.Network/virtualHubs/hubRouteTables', 'Hub-01', 'RT_VNet')]"
+      ],
+      "properties": {
+        "remoteVirtualNetwork": {
```

Comment:
format the JSON (e.g. SHIFT+ALT+F in VS Code)

Reply:
JSON is formatted already using SHIFT+ALT+F.